lambda-resume-builder/src/Components/Register/Register.js

```diff
@@ -1,175 +1,144 @@
-    [theme.breakpoints.up(400 + theme.spacing.unit * 3 * 2)]: {
-      width: 400,
-      marginLeft: 'auto',
-      marginRight: 'auto',
-    },
-  },
-  paper: {
-    marginTop: theme.spacing.unit * 8,
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
-    padding: `${theme.spacing.unit * 2}px ${theme.spacing.unit * 3}px ${theme.spacing.unit * 3}px`,
-  },
-  avatar: {
-    margin: theme.spacing.unit,
-    backgroundColor: theme.palette.secondary.main,
-  },
-  form: {
-    width: '100%', // Fix IE 11 issue.
-    marginTop: theme.spacing.unit,
+import Paper from '@material-ui/core/Paper'
+import withStyles from '@material-ui/core/styles/withStyles'
+import { NavLink } from 'react-router-dom'
+
+import { register as styles } from '../../MaterialUI/styles'
+
+
+
+
+const StyledButton = withStyles({
+  root: {
+    background: '#bb1333',
+    borderRadius: 3,
+    border: 0,
+    color: 'white',
+    height: 48,
+    padding: '0 30px',
+    boxShadow: '0 3px 5px 2px #D0CECF',
   },
-  submit: {
-    marginTop: theme.spacing.unit * 3,
+  label: {
+    textTransform: 'capitalize',
   },
-});
+})(Button)
 
-const StyledButton = withStyles({
-    root: {
-      background: '#bb1333',
-      borderRadius: 3,
-      border: 0,
-      color: 'white',
-      height: 48,
-      padding: '0 30px',
-      boxShadow: '0 3px 5px 2px #D0CECF',
-    },
-    label: {
-      textTransform: 'capitalize',
-    },
-  })(Button);
-
-  const lambdaLogo={
-    height: '40px',
-    width: '120px',
-    padding: '5px'
+const lambdaLogo = {
+  height: '40px',
+  width: '120px',
+  padding: '5px'
 }
 
-const Register = (props) =>  {
-  
-   
-  
-    const [state, setState]= React.useState(
+const Register = (props) => {
+
+
+
+  const [state, setState] = React.useState(
     {
-    role: "",
-    email: "",
-    password: ""})
-
-    const { classes } = props;
-    
-   function handleChange(event) {
-    setState({...state, [event.target.name]:event.target.value});
+      role: "",
+      email: "",
+      password: ""
+    })
+
+  const { classes } = props
+
+  function handleChange(event) {
+    setState({ ...state, [event.target.name]: event.target.value })
   }
 
   const redirect = () => {
     console.log('Props', props)
-    window.location.pathname = '/dashboard';
-    
-   }
+    window.location.pathname = '/dashboard'
+
+  }
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    props.register(state)
+    setTimeout(() => redirect(), 1000)
 
- const handleSubmit = async event => {
-    event.preventDefault();
-    await props.register(state);
-    setTimeout( () => redirect(), 1000)
 
-    
   }
 
-  
+
   return (
     <main className={classes.main}>
-     
+
       <CssBaseline />
-      <p style={{width: '100%'}}>
+      <p style={{ width: '100%' }}>
       </p>
       <Paper className={classes.paper} >
-        
-        <img style={lambdaLogo} src={Logo}/>
+
+        <img alt='' style={lambdaLogo} src={Logo} />
```

Comment:
Maybe a little more time actually providing a useful alt tag, you must be a real hit amongst the visually impaired community.